### PR TITLE
Added a section about building and testing to the readme

### PR DESCRIPTION
**Pull request by alexanderkjall**
Added a small section on how to build and test the checks locally,

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,4 +50,21 @@ Currently there's no vendoring provided for this project.
...
+
+In every check directory there is an example configuration file called `local.toml.example`, the checks reads a
+file named `local.toml` by default, so by copying that file you can do a local test of your code before you commit.
+
+Full example to build and test run one check:
+```
+cd cmd/vulcan-drupal
+go build
+cp local.toml.example local.toml
+./vulcan-drupal
```
Comment (gguillemas) [inline README.md:69]: I believe for the local checks to work with the provided file, the `-t` flag is required. I think this line should be:
```./vulcan-drupal -t```

**Review by gguillemas — CHANGES_REQUESTED** [review]
Thanks for this information, I think it will be very useful to new contributors!

I have suggested a small correction that I think is necessary for the instructions to work.

It may be worth noting that this method of testing will be insufficient for checks that are not completely self-contained in the Go code, which will need to run in their provided Docker environment. This can be achieved by using the `vulcan-build-images -r` command provided in the `vulcan-checks-bsys` project that you already mentioned.

File: README.md (hunk 1 above)
Comment (alexanderkjall) [reply README.md]: info added in new commit, thanks for feedback :)

File: README.md [hunk 2]
```diff
@@ -50,4 +50,26 @@ Currently there's no vendoring provided for this project.
...
+The checks that are not self-contained, and instead relay on external binaries, can be locally tested with
+`vulcan-checks-bsys`. By running `vulcan-build-images -r PATH_TO_THE_CHECK_DIR` it will create a docker image and
+run it, feeding the check the input from local.toml.
+
+Full example to build and test run one check:
+```
+cd cmd/vulcan-drupal
+go build
+cp local.toml.example local.toml
+./vulcan-drupal -t
```
Comment (julianvilas) [inline README.md:74]: I suggest adding here also the alternative option:
```
go run github.com/adevinta/vulcan-checks-bsys/cmd/vulcan-build-images@master -r ../vulcan-drupal 
```

File: README.md [hunk 3]
```diff
@@ -50,4 +50,26 @@ Currently there's no vendoring provided for this project.
...
+
+But it's possible to build each one of the checks with `go build`.
+
+In every check directory there is an example configuration file called `local.toml.example`, most checks reads a
+file named `local.toml` if you pass the `-t` parameter, so by copying that file you can do a local test of your code
+before you commit.
+
+The checks that are not self-contained, and instead relay on external binaries, can be locally tested with
+`vulcan-checks-bsys`. By running `vulcan-build-images -r PATH_TO_THE_CHECK_DIR` it will create a docker image and
+run it, feeding the check the input from local.toml.
```
Comment (julianvilas) [inline README.md:67]: Suggestion: `s/check the input/check with the input/`

File: README.md [hunk 4]
```diff
@@ -50,4 +50,26 @@ Currently there's no vendoring provided for this project.
...
+
+This project is primarily built using the [vulcan-checks-bsys](https://github.com/adevinta/vulcan-checks-bsys) project.
+
+But it's possible to build each one of the checks with `go build`.
+
+In every check directory there is an example configuration file called `local.toml.example`, most checks reads a
+file named `local.toml` if you pass the `-t` parameter, so by copying that file you can do a local test of your code
+before you commit.
+
+The checks that are not self-contained, and instead relay on external binaries, can be locally tested with
```
Comment (julianvilas) [inline README.md:65]: Typo: `s/relay/rely/`

**Review by julianvilas — COMMENTED** [review]
Just some extra minor suggestions :)

File: README.md (hunk 2 above)
Comment (alexanderkjall) [reply README.md:74]: `go run` doesn't seem to support that, I'm getting:
```
$ go run github.com/adevinta/vulcan-checks-bsys/cmd/vulcan-build-images@master -r ../vulcan-drupal 
package github.com/adevinta/vulcan-checks-bsys/cmd/vulcan-build-images@master: can only use path@version syntax with 'go get' and 'go install' in module-aware mode
```

File: README.md (hunk 3 above)
Comment (alexanderkjall) [reply README.md]: fixed

File: README.md (hunk 4 above)
Comment (alexanderkjall) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (julianvilas) [reply README.md:74]: Yeah! it's new from Go 1.17 :)

```
$ cd cmd/vulcan-drupal
$ go run github.com/adevinta/vulcan-checks-bsys/cmd/vulcan-build-images@master -r ../vulcan-drupal
/var/folders/rm/df56ygm54w5_prg9tk2xfp_c0000gn/T/go-build2598715718/b001/exe/vulcan-build-images: main.go:447: Reading manifest for dir ../vulcan-drupal
/var/folders/rm/df56ygm54w5_prg9tk2xfp_c0000gn/T/go-build2598715718/b001/exe/vulcan-build-images: main.go:470: Running go build for dir ../vulcan-drupal
/var/folders/rm/df56ygm54w5_prg9tk2xfp_c0000gn/T/go-build2598715718/b001/exe/vulcan-build-images: main.go:453: Building image for dir ../vulcan-drupal
/var/folders/rm/df56ygm54w5_prg9tk2xfp_c0000gn/T/go-build2598715718/b001/exe/vulcan-build-images: main.go:465: Docker image built, build log:
...
```

For older versions you should use `go install` or `go get`, depending on the version also.

File: README.md (hunk 4 above)
Comment (julianvilas) [reply README.md]: ty!

File: README.md (hunk 2 above)
Comment (alexanderkjall) [reply README.md:74]: Added information about `go run` together with the caveat about go version

Comment (julianvilas) [reply README.md:74]: very cool, thx! :)

**Review by julianvilas — APPROVED** [review]
Looks Super Good To Me! :)

**Review by gguillemas — APPROVED** [review]
I apologize, I was not aware that my initial review was blocking the merge. It looks really good!

**Pull request merged**